### PR TITLE
https://issues.redhat.com/browse/ACM-9002 01/16/24 MCE 2.4.3

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-9002

### CODE REVIEW

File: clusters/release_notes/errata.adoc [hunk 1]
```diff
@@ -5,6 +5,14 @@ For {mce-short}, the Errata updates are automatically applied when released.
...
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
+== Errata 2.4.3
+
+* Delivers updates to one or more of the product container images.
+
+* Fixes an issue where the node regions and zones were incorrectly displayed from the _Topolgy_ view. (link:https://issues.redhat.com/browse/ACM-8996[ACM-8996]).
+
+* Fixes an issue where deprecated node labels were being used. (link:https://issues.redhat.com/browse/ACM-9002[ACM-9002]).
+
```
Comment (swopebe) [inline clusters/release_notes/errata.adoc:15]: Let's get rid of "being" here and you can just say, "...were used."

(concise, active)

**Review by swopebe — APPROVED** [review]
Thanks for this.

File: clusters/release_notes/errata.adoc (hunk 1 above)
Comment (dockerymick) [reply clusters/release_notes/errata.adoc:15]: Gotchu! Thanks for the reminder! Updating now

File: clusters/release_notes/errata.adoc [hunk 2]
```diff
@@ -5,6 +5,14 @@ For {mce-short}, the Errata updates are automatically applied when released.
 
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user. 
 
+== Errata 2.4.3
+
+* Delivers updates to one or more of the product container images.
+
+* Fixes an issue where the node regions and zones were incorrectly displayed from the _Topolgy_ view. (link:https://issues.redhat.com/browse/ACM-8996[ACM-8996]).
+
+* Fixes an issue where deprecated node labels were being used. (link:https://issues.redhat.com/browse/ACM-9002[ACM-9002]).
```
Comment (dockerymick) [inline clusters/release_notes/errata.adoc]:
```suggestion
* Fixes an issue where deprecated node labels were used. (link:https://issues.redhat.com/browse/ACM-9002[ACM-9002]).
```

**Comment by dockerymick** [comment]
Thanks for your review and approval Brandi!

**Pull request merged**